src/gpm-control.c

```diff
@@ -233,6 +243,35 @@ gpm_control_suspend (GpmControl *control, GError **error)
 		}
 	}
 
+#ifdef WITH_LIBSECRET
+	/* we should perhaps lock keyrings when sleeping #375681 */
+	lock_libsecret = g_settings_get_boolean (control->priv->settings, GPM_SETTINGS_LOCK_KEYRING_SUSPEND);
+	if (lock_libsecret) {
+    libsecret_cancellable = g_cancellable_new();
+    secretservice_proxy = secret_service_get_sync(SECRET_SERVICE_LOAD_COLLECTIONS,
+                    libsecret_cancellable,
```

Comment:
Indents .....
Please us a space before bracket.

Reply:
How did you want the long function call on 251 wrapped?  I couldn't find any reference to it on the wiki and the examples I saw in the code seemed inconsistent.

Reply:
I just updated the 2 commits for you. See next post.